pkg/ocm/helpers.go

```diff
@@ -813,6 +813,14 @@ func ValidateOperatorRolesMatchOidcProvider(awsClient aws.Client,
 			return weberr.Errorf("Operator role '%s' does not have trusted relationship to '%s' issuer URL",
 				roleARN, parsedUrl.Host)
 		}
+		hasManagedPolicies, err := awsClient.HasManagedPolicies(roleARN)
+		if err != nil {
+			return err
+		}
+		if hasManagedPolicies {
+			// Managed policies should be compatible with all versions
```

Comment:
I am not a fan of the word `should`  is there any kind of validations we can do here to ensure they are compatible?

Reply:
Never mind, I just read the ticket, we use a tag to check compatibility, and the tag does not exist on managed policies. 
That said I still feel we need some method to validate these policies

Reply:
Given this solves the issue in the ticket, lets merge it.

Reply:
To validate we would need to start going through all permission actions and have a list that matches each functionality

It is possible, just not something being done at the moment or expected to be done

Reply:
Ill make note, and create a ticket for this, it be nice to validate at the granular level of permissions and not just on a tag

Reply:
The `should` refers here to that the managed policies are owned by AWS and we are working under the assumption they are always up to date.
Unlike unmanaged policies, where the user has to upgrade.